.gitignore

```diff
@@ -26,11 +26,14 @@ build
 
 # misc
 .DS_Store
-.env*
```

Comment:
removing .env* from git ignore is bad idea
someone migh accidentally commit their personal .evn which might have private keys and such